addons/default/visiosoft/advs-module/src/AdvsModulePlugin.php

```diff
@@ -136,6 +136,12 @@ function ($array) {
                     return null;
                 }
             ),
+            new \Twig_SimpleFilter(
+                'json_decode',
+                function ($array) {
+                    return json_decode($array, true);
```

Comment:
twig json_decode not present by default?

Reply:
there is encode. No decode @muammertop

Reply:
i used yesterday @spektra2147

Reply:
ok now I'm checking again.

Reply:
can be used as a function.  `json_decode(test_json,true)`